ui-lib/src/main/res/ui/advanced_settings/values/strings.xml

```diff
@@ -2,6 +2,7 @@
     <string name="advanced_settings_backup_wallet">Recovery phrase</string>
     <string name="advanced_settings_export_private_data">Export private data</string>
     <string name="advanced_settings_choose_server">Choose a server</string>
+    <string name="advanced_settings_coinbase">Buy ZEC with</string>
```

Comment:
We could leverage `ZcashCurrency` in this case to distinguish between Testnet TAZ and Mainnet ZEC currencies.